services/ui-src/src/components/app/AppRoutes.test.tsx

```diff
@@ -3,36 +3,43 @@ import { act } from "react-dom/test-utils";
 import { Router } from "react-router-dom";
 import { createMemoryHistory } from "history";
 import { AppRoutes } from "components";
-import { useUser } from "utils";
+import { useUser, UserProvider } from "utils";
 import { mockAdminUser, mockStateUser } from "utils/testing/setupJest";
 
 jest.mock("utils/auth/useUser");
 const mockedUseUser = useUser as jest.MockedFunction<typeof useUser>;
 
 const appRoutesComponent = (history: any) => (
   <Router location={history.location} navigator={history}>
-    <AppRoutes />
+    <UserProvider>
+      <AppRoutes />
+    </UserProvider>
   </Router>
 );
 
 let history: any;
+const tempScroll = window.HTMLElement.prototype.scrollIntoView;
 
-describe("Test AppRoutes for admin-specific routes", () => {
+describe("brax Test AppRoutes for admin-specific routes", () => {
   beforeEach(async () => {
+    window.HTMLElement.prototype.scrollIntoView = function () {};
     mockedUseUser.mockReturnValue(mockAdminUser);
     history = createMemoryHistory();
     history.push("/admin");
     await act(async () => {
       await render(appRoutesComponent(history));
     });
   });
+  afterEach(async () => {
+    window.HTMLElement.prototype.scrollIntoView = tempScroll;
+  });
   test("/admin is visible for admin user", async () => {
     const currentPath = history.location.pathname;
     expect(currentPath).toEqual("/admin");
   });
 });
 
-describe("Test AppRoutes for non-admin-specific routes", () => {
+describe("brax Test AppRoutes for non-admin-specific routes", () => {
```

Comment:
I don't know that Brax needs to sign his tests

Reply:
lol sometimes i like to leave a calling card, ok?